admin/base/views.py

```diff
@@ -28,3 +34,43 @@ def password_reset_confirm_custom(request, **kwargs):
             user.confirmed = True
             user.save()
     return response
+
+
+class GuidFormView(FormView):
+    form_class = GuidForm
+    template_name = None
+    object_type = None
+
+    def get_guid_object(self):
+        raise NotImplementedError
+
+    def __init__(self):
+        self.guid = None
+        super(GuidFormView, self).__init__()
+
+    def get(self, request, *args, **kwargs):
+        try:
+            return super(GuidFormView, self).get(request, args, kwargs)
+        except AttributeError:
```

Comment:
This will swallow AttributeErrors that might not be related to fetching GUIDs, leading to bugs that are hard to find. Why do you expect an `AttributeError` here? https://github.com/CenterForOpenScience/osf.io/pull/4960/files#diff-d0d8d8e2e4760424d31eacb32ecdeb29R63 

It would make sense for concrete implementations of `get_guid_object` to raise exceptions or return an error response rather than handling it in this class.